src/JsonViewer/stories.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { storyDocsParameters } from '../utils';
+import { JsonViewer } from './index';
+
+export default {
+  title: 'Design System/JsonViewer',
```

Comment:
I would go with
`Design System/Viewer/Json`

in case a day we have other viewers (such as a pdf viewer for report viewing).